cmake/features.cmake

```diff
@@ -122,8 +122,7 @@ else()
     set(protoc_available ON)
 endif()
 
-ie_dependent_option(NGRAPH_ONNX_IMPORT_ENABLE "Enable ONNX importer" ON "protoc_available" OFF)
-ie_dependent_option(NGRAPH_ONNX_FRONTEND_ENABLE "Enable ONNX FrontEnd" OFF "NGRAPH_ONNX_IMPORT_ENABLE" OFF)
+ie_dependent_option(NGRAPH_ONNX_IMPORT_ENABLE "Enable ONNX FrontEnd" ON "protoc_available" OFF)
```

Comment:
may be better to keep  `NGRAPH_ONNX_FRONTEND_ENABLE` name since importer is obsolete name? currently we operate with frontends